### PR TITLE
feat: Added bulk delete to version change view

### PR DESCRIPTION
**Pull request by polyccon**
## Description

*- Initial draft on adding functionality to bulk delete to versions djangocms versioning
*- More information required on the ticket that needs to be a user story - see link below 

## Related resources

https://github.com/orgs/django-cms/projects/2/views/3?pane=issue&itemId=24538627
* #...
* #...

## Checklist

* [ ] I have opened this pull request against ``master``
* [ ] I have added or modified the tests when changing logic
* [ ] I have followed [the conventional commits guidelines](https://www.conventionalcommits.org/) to add meaningful information into the changelog
* [ ] I have read the [contribution guidelines ](https://github.com/django-cms/django-cms/blob/develop/CONTRIBUTING.rst) and I have joined #workgroup-pr-review on 
[Slack](https://www.django-cms.org/slack) to find a “pr review buddy” who is going to review my pull request.

### CODE REVIEW

**Comment by fsbraun** [comment]
@polyccon I updated your PR. People keep asking for this feature! I guess you were 80% there. I hope you do not mind, that I added some stuff to finish this.

File: tests/test_admin.py [hunk 1]
```diff
@@ -2652,6 +2652,81 @@ def test_change_view_action_compare_versions_three_selected(self):
...
+                "post": "yes",
+            }
+            response = self.client.post(endpoint, data, follow=True)
+
+        self.assertEqual(response.status_code, 200)
+        self.assertEqual(PollContent._base_manager.all().count(), 1 + 4)
+
+    @patch("djangocms_versioning.conf.ALLOW_DELETING_VERSIONS", True)
+    def test_bulk_delete_action_confirmation(self):
+        version = factories.PollVersionFactory(state=ARCHIVED)
```
Comment (fsbraun) [inline tests/test_admin.py:2713]:
```suggestion
        version = factories.PollVersionFactory(state=constants.ARCHIVED)
```

File: djangocms_versioning/admin.py [hunk 2]
```diff
@@ -927,6 +920,44 @@ def compare_versions(self, request, queryset):
...
+        """
+        # Do not allow deleting single version objects. Use discard instead.
+        forbidden = queryset.filter(state__in=(PUBLISHED, DRAFT))
+        if forbidden.exists():
+            self.message_user(
+                request,
+                _("Draft or published versions cannot be deleted. First unpublish or use discard for drafts."),
+                messages.ERROR
+            )
+            return
```
Comment (fsbraun) [inline djangocms_versioning/admin.py:943]:
```suggestion
            return None
```

**Review by fsbraun — APPROVED** [review]
@polyccon Thanks so much for getting this started. It's been a feature that people have kept requesting. Great that we can finish this now!

**Comment by polyccon** [comment]
Nice work @fsbraun thank you for your hard work to bring it over the finish line!!!!!!

**Pull request merged**